packages/groqd/src/sanityImage.ts

```diff
@@ -30,7 +30,7 @@ const hotspotFields = {
 };
 
 const refBase = {
-  _key: schemas.string().nullable(),
+  _key: schemas.string().optional(),
```

Comment:
There's a small problem with this. The `refBase` variable gets used in `toGrab` – and `toGrab` is used inside of a `.grab` projection call. This means that the query generated has something like `{ _key }` in it. If you request a field that's absent, the result is `null` – and not `undefined`. Therefore this change would likely throw an error for missing `_key`s.

Reply:
Ok I understand, any idea how to fix the error I'm having for example?